scripts/makeDatabase.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-plusplus */
+const fs = require('fs');
+const { users: databaseUsers, roles: databaseRoles, events: databaseEvents } = require('./webDatabase');
+
+const userToId = () => {
+    const mappedUsers = {};
+    databaseUsers.map((user, index) => {
```

Comment:
This is what I mean. Here: 
```suggestion
    databaseUsers.forEach((user, index) => {
```